packages/code-du-travail-api/src/server/routes/search/index.js

```diff
@@ -43,32 +45,40 @@ router.get("/search", async ctx => {
   // we add 1 to maxResults in case we have a snippet document in the results
```

Comment:
If the +1 is not necessary anymore we might want to remove that comment too

Reply:
exact!

Reply:
Does it seems to be fixable ? because there is other code that rely on snippet. We should decide if we remove all the code related to snippet (from ES queries to frontend) or we should open an issue to fix it later